### ISSUE TITLE
Selecting items in MultiSelect using keyboard navigation for "SelectNextItem" or "SelectPreviousItem" selects multiple items

### ISSUE THREAD

**Issue by aleksandarevangelatov** [telerik/kendo-ui-core#5283]
### Bug report

Selecting items in MultiSelect using keyboard navigation for "SelectNextItem"or "SelectPreviousItem" results in multiple items selection. 

### Reproduction of the problem

1.  Open the MultiSelect [Keyboard navigation demo]((https://demos.telerik.com/kendo-ui/multiselect/keyboard-navigation)
2.  select two consecutive items with SHIFT+DOWN-ARROW (e.g. "Small" and "Medium")
3.  use the arrow keys to move the focus further down (e.g. to "X-Large")
4.  press SHIFT+DOWN-ARROW again

### Current behavior

All items get selected - Small, Medium, Large, X-Large and 2X-Large

### Expected/desired behavior

Only three items should be selected - Small, Medium and 2X-Large

### Environment

* **Kendo UI version:** 2023.3.1114
* **Browser:** [all]

**Comment by VicTachev** [comment]
The general guidance is to use Shift when selecting multiple consecutive items. If some items should be skipped - selection is performed with Ctrl+Space.
